decidim-proposals/app/packs/stylesheets/decidim/proposals/proposals.scss

```diff
@@ -374,31 +400,3 @@
     @apply lg:col-start-2 lg:col-span-10;
   }
 }
-
-#form-merge-proposals {
```

Comment:
#form-merge-proposals {
  .custom-inputs {
    input {
      @apply mt-0;
    }
  }

  .upload-modal__files-container {
    label {
      @apply mb-0;
    }

    button {
      @apply mt-0;
    }
  }
}

.form__wrapper,
.form-proposals-merge-styles {
  .proposal-field-custom-style {
    & label:not(&-checkbox-label),
    &-block label:not(&-checkbox-label) {
      @apply font-normal text-sm;
    }
  }
}